### PR TITLE
Feature/update error message

### PR DESCRIPTION
**Pull request by SarthakSingh31**

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -21,17 +21,17 @@
...
     <string name="not_signed_in">Welcome</string>
     <string name="sign_in">Sign in</string>
-    <string name="you_are_not_signed_in">You are not signed in</string>
-    <string name="sign_in_error">Unable to Load Sign-In Info</string>
-    <string name="sign_in_error_desc">There was an error reading the last login statue. Please clean the app storage and launch again. This message needs to be updated by ux</string>
-    <string name="no_internet">No Internet</string>
-    <string name="no_internet_desc">No Internet</string>
+    <string name="you_are_not_signed_in">Please sign in to continue.</string>
+    <string name="sign_in_error">Signed Out</string>
+    <string name="sign_in_error_desc">You have been signed out, likely due to an application error or prolonged inactivity.\nSign in to continue.</string>
```
Comment (DDaiJ) [inline app/src/main/res/values/strings.xml:26]: I'm a bit confused why we have these two strings same as line 35 & 36 but with different names.

Comment (SarthakSingh31) [reply app/src/main/res/values/strings.xml:26]: Internally they represent different errors but to make it easier for the user they have the same description.

Comment (DDaiJ) [reply app/src/main/res/values/strings.xml:26]: Then maybe we can use a more generic name and combine the two?

Comment (abztrakt) [reply app/src/main/res/values/strings.xml:26]: I think it's appropriate to keep them represented separately internally, in this case.

**Review by abztrakt — APPROVED** [review]

**Pull request merged**